jujugui/static/gui/src/app/components/profile/profile.js

```diff
@@ -43,25 +39,51 @@ class Profile extends React.Component {
 Profile.sectionsMap = new Map([
   ['models', {
     label: 'Models',
-    component: 'Models'
+    getComponent: context => {
+      return (
+        <juju.components.ProfileModelList
+          acl={context.props.acl}
+          addNotification={context.props.addNotification}
+          changeState={context.props.changeState}
+          facadesExist={context.props.facadesExist}
+          destroyModels={context.props.destroyModels}
+          listModelsWithInfo={context.props.listModelsWithInfo}
+          switchModel={context.props.switchModel}
+          userInfo={context.props.userInfo} />);
+    }
   }],
   ['charms', {
     label: 'Charms',
-    component: 'Charms'
+    getComponent: context => 'Charms'
```

Comment:
Rather than context (very generic/ it could be anuthing) isn't this the actual component instance? So why not comp, or compInstance?